seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/operation/DeployTaskOperation.java

```diff
@@ -40,9 +40,9 @@ public DeployTaskOperation(@NonNull Data taskImmutableInformation) {
     }
 
     @Override
-    protected NonCompletableFuture<?> doRun() throws Exception {
-        SeaTunnelServer server = getService();
-        return server.getTaskExecutionService().deployTask(taskImmutableInformation);
+    protected PassiveCompletableFuture<?> doRun() throws Exception {
+        TaskExecutionService taskExecutionService = getService();
```

Comment:
Does `TaskExecutionService` is an hazelcast service? Can use `getService()` to get it?

Reply:
Yes, `TaskExecutionService is an hazelcast service` is right.